qa/rspec/commands/redhat.rb

```diff
@@ -23,11 +23,15 @@ def package_for(filename, base=ServiceTester::Base::LOCATION)
     def install(package, host=nil)
       hosts  = (host.nil? ? servers : Array(host))
       errors = []
+      exit_status = 0
       at(hosts, {in: :serial}) do |_host|
         cmd = sudo_exec!("yum install -y  #{package}")
+        exit_status += cmd.exit_status
         errors << cmd.stderr unless cmd.stderr.empty?
       end
-      raise InstallException.new(errors.join("\n")) unless errors.empty?
+      if exit_status > 0 
```

Comment:
I see your intent in doing this. I guess the only downside is that anything that is written to stderr by commands that terminate with status == 0 won't be logged/registered/shown anywhere.

I'd perhaps make the errors a list of `{"host" => host, "command" => cmd, "exit_status" => 0, "stderr" => cmd.stderr }` and raise an InstallException if either an exit_status isn't 0 or stderr isn't empty? not sure...thoughts?